pagopa/io-app
# fix: [IOCOM-1614] Update iOS product name to "IO" from "ItaliaApp"

## Short description
updated iOS product name in order for system popups to display the correct naming
|BEFORE|AFTER|
|-|-|
|<img src="https://github.com/user-attachments/assets/b82f00cc-4cfe-4f61-a04d-6c679d8fbdfd" width="250"/>|<img src="https://github.com/user-attachments/assets/d9926ca8-667e-4e86-859c-b739203087a4" width="250"/>

## List of changes proposed in this pull request
updated product naming wherever necessary

## How to test
remove both the `Build` and `DerivedData` folders under `ios`, then, while running the dev-server in the FIMS branch, run a fims auth flow and make sure the popup displays the correct product name, as shown in the screenshot.
